iscript/src/iscript/constants.py

```diff
@@ -18,7 +18,7 @@
     "mozillavpn": {
         "designated_requirements": """=designated => certificate leaf[subject.OU] = "%(subject_ou)s" """,
         "sign_dirs": ("MacOS", "Frameworks", "Resources"),
-        "skip_dirs": ("MozillaVPNLoginItem.app",),
+        "skip_dirs": ("MozillaVPNLoginItem.app", "utils"),
```

Comment:
We should probably at least file a follow-up issue to fix things so that `skip_dirs` normalizes all paths to be relative to the root of the package. Otherwise, this will skip all dirs named `utils` and there will be no way to specify which one.

Reply:
https://mozilla-hub.atlassian.net/browse/RELENG-895 :)